avaframe/ana3AIMEC/ana3AIMECCfg.ini

```diff
@@ -9,6 +9,10 @@
 # width of the domain around the avalanche path in [m]
 domainWidth = 600
 
+# The cell size for the new (s,l) raster is automatically computed from the input dem (leave the following field empty)
+# It is possible to force the cell size to take another value, then specify this new value bellow.
```

Comment:
```suggestion
# It is possible to force the cell size to take another value, then specify this new value below, otherwise leave empty (default).
```